.travis.yml

```diff
@@ -4,7 +4,6 @@ os:
   - linux
   - osx
 julia:
-  - 1.0
   - 1
```

Comment:
Could you change the 1.0 to 1.3, such that we always test on the oldest julia version we claim we support, as well as the latest 1.x (1), and nightly.

Same for AppVeyor.

Reply:
done